spec/student_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../student'
+
+describe Student do
+  before(:each) do
+    @student = Student.new('microverse', 27, 'MhdezCoronado')
+  end
```

Comment:
- [ ] try to be as thorough as possible when testing, for example looking at parent permission make sure to test positive and negative cases